test/test_util.ts

```diff
@@ -0,0 +1,21 @@
+import * as fs from "fs"
+import { ExampleParser } from "../src/index"
+
+export function parseFile (txtPath: string) {
+    // Load a text to test from the FS
+    let text: string = ''
+    try {
+        text = fs.readFileSync(txtPath, 'utf8').toString()
+    } catch (e: any) {
+        console.log('Error:', e.stack)
+    }
+    // Initialize the parser
+    let parser_inst: ExampleParser = new ExampleParser(text)
+    return parser_inst.mapLines()
```

Comment:
Is there a reason to continue test execution when file reading failed?

I'd consider to remove error handling and leave it to the test framework.